widgets/mem_freebsd.lua

```diff
@@ -0,0 +1,67 @@
+----------------------------------------------
```

Comment:
I have once written the same widget that looks almost the same but I added swap, too. Have a look at it [here](https://bitbucket.org/mutluyum/vicious_bsd/src/c524e4ada9960876f0914af9748945078859b156/widgets/mem_freebsd.lua?at=default&fileviewer=file-view-default).